src/components/Button/ButtonAddPostTopic.js

```diff
@@ -35,6 +36,10 @@ export default ButtonAddPostTopic;
 
 const styles = StyleSheet.create({
   container: {
+    position: 'absolute',
+    bottom: normalize(30),
+    right: 0,
+    zIndex: 99,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `zIndex` property is used to control stack order of elements. However, it's important to note that using high arbitrary values for `zIndex` can lead to maintainability issues in the future as it may become difficult to manage stacking contexts. If there are no other elements with a `zIndex` set, consider starting from 1.
```diff
-    zIndex: 99,
+    zIndex: 1,
```
Also, the `backgroundColor` property seems to have an invalid value `'#0000'`. This might be a typo or an oversight. Please ensure to provide a valid color code.
```diff
-    backgroundColor: '#0000'
+    backgroundColor: '#000000'
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->